src/drivers/imu/mpu6000/mpu6000.cpp

```diff
@@ -738,7 +738,7 @@ MPU6000::init()
 
 	/* measurement will have generated a report, publish */
 	_accel_topic = orb_advertise_multi(ORB_ID(sensor_accel), &arp,
-					   &_accel_orb_class_instance, (is_external()) ? ORB_PRIO_MAX : ORB_PRIO_HIGH);
+                       &_accel_orb_class_instance, (is_external()) ? ORB_PRIO_DEFAULT : ORB_PRIO_HIGH);
```

Comment:
Why is this necessary?

Reply:
This is just an experimental revision for the sake of convenience that we can block the mpu6000 by priority swapping